LibreNMS/Util/ModuleTestHelper.php

```diff
@@ -191,7 +191,7 @@ private function collectOids($device_id)
         $collection_output = preg_replace('/\033\[[\d;]+m/', '', $collection_output);
 
         // extract snmp queries
-        $snmp_query_regex = '/SNMP\[.*snmp(?:bulk)?([a-z]+)\' .+(udp|tcp|tcp6|udp6):(?:\[[0-9a-f:]+\]|[^:]+):[0-9]+\' \'(.+)\']/';
+        $snmp_query_regex = '/^SNMP\[\'.*snmp(?:bulk)?(walk|get|getnext)\' .+\'(udp|tcp|tcp6|udp6):(?:\[[0-9a-f:]+\]|[^:]+):[0-9]+\' \'(.+)\'\]$/m';
```

Comment:
Hmm, I wonder if the start and end tags might be a problem, 
not all output is "clean", here is a line from a random poller run:
`ifMtu SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OQUst' '-m' 'IF-MIB' '-M' '/opt/librenms/mibs' 'udp:HOSTNAME:161' 'ifMtu']`

Reply:
@Jellyfrog that looks like a separate bug :)